src/test_typing_extensions.py

```diff
@@ -6617,6 +6617,21 @@ def test_allow_default_after_non_default_in_alias(self):
         a4 = Callable[[Unpack[Ts]], T]
         self.assertEqual(a4.__args__, (Unpack[Ts], T))
 
+    def test_generic_with_broken_eq(self):
```

Comment:
as a short-term fix until we fix the CPython bug:

```suggestion
    @skip_if_py313_beta_1
    def test_generic_with_broken_eq(self):
```